### PR TITLE
fix: added-the-word-Development-to-front-end-libraries

### LINKED ISSUE #42502: Change cert name to "Front End Development Libraries" on settings page
We renamed the "Front End Libraries" cert to "Front End Development Libraries" a while back. It never got updated in a few spots on the settings page...
Screen Shot 2021-06-15 at 6 58 30 AM
Screen Shot 2021-06-15 at 6 58 13 AM

We should update the name in these two spots

### PR DESCRIPTION
**Pull request by NoorFakhry**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #42502

### CODE REVIEW

**Comment by raisedadead** [comment]
Hi, @NoorFakhry Thanks for the PR. 

Just a heads up that we have a prior PR that addresses the same issue (see #42511). We are waiting for a response from the PR creator about expanding tests. We would hold on to this PR unless they get back to us with the requested changes.

Thanks for your patience and understanding meanwhile.

**Comment by NoorFakhry** [comment]
@raisedadead Thank You for the reply, sorry I didn't pay attention to that, since this is my first open-source contribution, so I am still learning :D

**Comment by raisedadead** [comment]
Hey @NoorFakhry 

It's all good and welcome to contributing to open-source. We appreciate it.

You absolutely do not pay attention to other PRs, although it might be helpful from time to time. It's a coincidence that someone else was working on the same issue and happens all the time. 

We are just keeping this on hold while we get a reply from them. 

Meanwhile, we strongly encourage you to take a look at open help-wanted issues. Let us know if you need further help in the chat.

**Comment by raisedadead** [comment]
Hi @NoorFakhry 

The prior PR was closed as requested by the OP. We have since enabled this PR for QA. Please take a look at the original issue thread for additional information if needed for you to work on this PR.

There are two approaches here that you can take, depending on your interest. Ideally, we would want the second approach because that will cover all the bases. We will let you decide.

1. Only make changes to the settings page and add/update a test to go with it. Someone else (most likely @ShaunSHamilton) will look into updating the bits in the curriculum and the tests to go with them.

2. (Ideal & what we would love to have) Update all the files and tests as mentioned in the original issue thread along with the settings page.

Let us know, and we are available in the chat to guide you through it.

**Comment by NoorFakhry** [comment]
Hi @raisedadead
I will start working on the issue, and I will do it by the second approach, If I am late don't worry I am just grasping how I am going to do it, But I will try to be as quickly as I can

**Comment by NoorFakhry** [comment]
Hey @raisedadead, Can you tell me is this okay, did it pass the tests?

File: client/src/pages/learn/front-end-libraries/front-end-libraries-projects/index.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Introduction to the Front End Libraries Projects
-block: Front End Libraries Projects
-superBlock: Front End Libraries
+title: Introduction to the Front End c Libraries Projects
+block: Front End v Libraries Projects
```
Comment (nhcarrigan) [inline client/src/pages/learn/front-end-libraries/front-end-libraries-projects/index.md:3]: What are the extra letters here?

Comment (NoorFakhry) [reply client/src/pages/learn/front-end-libraries/front-end-libraries-projects/index.md:3]: Sorry, I'll fix them

Comment (NoorFakhry) [reply client/src/pages/learn/front-end-libraries/front-end-libraries-projects/index.md]: Is it good now?

**Review by raisedadead — CHANGES_REQUESTED** [review]
I don't think you need to install or touch any dependencies. Please revert those changes.

Also, you would be needing to update the tests.

**Review by raisedadead — COMMENTED** [review]
Nice work so far, now you should be able to work thru the tests.

**Comment by NoorFakhry** [comment]
@raisedadead Thank you, I will start working on the tests shortly

**Comment by NoorFakhry** [comment]
Hey, I apologize but I realized It would take me time to understand the project as a whole and understand the code and its flow, So feel free to close this PR and let somebody else work on the issue, I will start contributing again once I feel comfortable with the code and have a better understanding of the project.

**Comment by ojeytonwilliams** [comment]
No apology necessary, @NoorFakhry, this section of our code base is far too complicated.  We have plans (lead by @ShaunSHamilton) to make it more manageable, but they're still in the pipeline.

Thanks for your hard work on this.  I'll leave this open, but as a draft, so that other contributors can benefit from seeing what you've done so far.

**Comment by raisedadead** [comment]
@ShaunSHamilton I have gone ahead and assigned this to you. I am hoping you are able to take care of this along with the other cert renaming.

**Comment by ShaunSHamilton** [comment]
@NoorFakhry Thank you, for this PR. I have accepted this as is, and just merged it into a work-in-progress branch on the `freeCodeCamp` repository so it can be worked off of without disturbing your fork.

**Comment by NoorFakhry** [comment]
@ShaunSHamilton Thank you, and as I promised I will work on understanding the codebase more deeply and hopefully do more valuable contributions in the future .

**Pull request merged**